### ISSUE TITLE
Interfaces with named getters inheriting from interfaces with named getters

### ISSUE THREAD

**Issue by domenic** [heycam/webidl#833]
The Web IDL spec says

> If an object implements more than one interface that defines a given special operation, then it is undefined which (if any) special operation is invoked for that operation.

However we have on the platform

```webidl
[Exposed=Window]
interface HTMLFormControlsCollection : HTMLCollection {
  // inherits length and item()
  getter (RadioNodeList or Element)? namedItem(DOMString name); // shadows inherited namedItem()
};

[Exposed=Window, LegacyUnenumerableNamedProperties]
interface HTMLCollection {
  readonly attribute unsigned long length;
  getter Element? item(unsigned long index);
  getter Element? namedItem(DOMString name);
};
```

Thus instances of `HTMLFormControlsCollection` implement two interface that both defined named getters, and we're in some kind of undefined situation.

We should presumably instead update the Web IDL spec to use the most-derived operation.

The normative text in e.g. [LegacyPlatformObjectGetOwnProperty](https://heycam.github.io/webidl/#LegacyPlatformObjectGetOwnProperty) is pretty unclear right now, talking a lot about running steps but not specifying what interface to run them for.

**Comment by bzbarsky** [comment]
> We should presumably instead update the Web IDL spec to use the most-derived operation.

I agree.  That's what browsers implement in practice, I expect.  Certainly it's what Gecko implements.
